api/client/samples/similar_regions_Frechet/metric.py

```diff
@@ -0,0 +1,179 @@
+### This file describes the metric to be used when doing region similarity computation.
```

Comment:
This file seems an example sample, we may want to indicate that in the filename

Reply:
I guess one could say this is an example. However, some sort of metric description is required to run the thing and in most cases it will created from a subset of what is presented in this file (rather than typed from scratch), I am reluctant to change the file name.

Reply:
Above I am talking mostly about metric_description which is the second (longer) dictionary in this file. I guess one could split this into two but I don't like small file proliferation. Can still change if you have a strong opinion on this...